main.tf

```diff
@@ -1,213 +1,52 @@
 locals {
-  network_name                    = "${var.project_id}-vpc"
-  subnet_name                     = "${var.project_id}-${var.cluster_region}-01"
-  secondary_ranges_pods_name      = "${var.project_id}-${var.cluster_region}-01-pods"
-  secondary_ranges_services_name  = "${var.project_id}-${var.cluster_region}-01-services"
-  gke_name                        = "${var.project_id}-gke"
-  node_pools_name                 = "${var.project_id}-node-pool-main"
-  cloud_router_name               = "${var.project_id}-router"
-  ci_service_account_name         = "ci-service-account"
-  nginx_controller_namespace_name = "nginx-controller"
+  storage_service_account_name = "storage-service-account"
+  ci_service_account_name      = "ci-service-account"
 }
 
 data "google_project" "project" {
   project_id = var.project_id
 }
 
-resource "google_project_default_service_accounts" "deprivilege_default_service_account" {
-  project = var.project_id
-  action  = "DEPRIVILEGE"
-}
-
-resource "google_project_service" "enable_cloud_resource_manager_api" {
-  project = var.project_id
-  service = "cloudresourcemanager.googleapis.com"
-}
-
-resource "google_project_service" "enable_compute_api" {
-  project = var.project_id
-  service = "compute.googleapis.com"
-}
-
-resource "google_project_service" "enable_container_api" {
+resource "google_compute_address" "ingress_ip_address" {
+  name    = "${var.project_name}-${var.environment_name}-nginx"
```

Comment:
I suggest renaming this to `${var.project_name}-${var.environment_name}-nginx-ingress-controller`